features/step_definitions/common_steps.rb

```diff
@@ -193,3 +199,16 @@
 Then(/^I see #{MAYBE_VAR} as the page header context$/) do |value|
   first(:xpath, "//header//*[@class='context']").text.should  == normalize_whitespace(value)
 end
+
+Then /^I see the '(.*)' summary table filled with:$/ do |table_heading, table|
```

Comment:
Yeah, wasn't me that bit! I think this code may be a hangover from some old tests.